src/Microsoft.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -124,6 +138,7 @@ public AsymmetricSignatureProvider(SecurityKey key, string algorithm, bool willC
 
             _minimumAsymmetricKeySizeInBitsForSigningMap = new Dictionary<string, int>(DefaultMinimumAsymmetricKeySizeInBitsForSigningMap);
             _minimumAsymmetricKeySizeInBitsForVerifyingMap = new Dictionary<string, int>(DefaultMinimumAsymmetricKeySizeInBitsForVerifyingMap);
+            _ecdsaKeySizeInBitsMap = new Dictionary<string, int>(DefaultECDsaKeySizeInBitsMap);
```

Comment:
Do you need to make a copy of the original map? Are you altering it in some way? It looks like you're just looking for a key in it, not changing it, so you should be able to use the original without copying it.